vividus-plugin-web-app-to-rest-api/src/main/java/org/vividus/bdd/steps/integration/ResourceCheckSteps.java

```diff
@@ -160,9 +158,22 @@ private String getSelector(Element element)
         }
     }
 
-    private Optional<String> getElementAttribute(Element element, String attributeKey)
+    private static Optional<String> getHrefAttribute(Element element)
     {
-        return element.hasAttr(attributeKey) ? Optional.of(element.attr(attributeKey)) : Optional.empty();
+        String href = element.attr(HREF_ATTR);
+        if (!href.isEmpty())
+        {
+            if (EXCLUDE_PATTERN.equals(href))
+            {
+                return Optional.of(href);
```

Comment:
what is the point to validate URL with an empty URL fragment?

Reply:
it's filtered prior to resource validation